core/src/main/scala/com/softwaremill/react/kafka/commit/ConsumerCommitter.scala

```diff
@@ -14,7 +14,9 @@ import scala.language.postfixOps
 /**
  * This should become deprecated as soon as we remove KafkaActorSubscriber/KafkaActorPublisher and related stuff.
  */
-private[commit] class ConsumerCommitter[K, V](
+// Commenting out the package private, to make tests more distant from code (using separate package names). AKa270216
+//private[commit]
+class ConsumerCommitter[K, V](
```

Comment:
The fact that tests were accessing a package private class probably implies that something should be reviewed within the particular test, or the class.